db/migrate/20161128200400_update_quotes.rb

```diff
@@ -0,0 +1,11 @@
+class UpdateQuotes < ActiveRecord::Migration
+  def change
+    change_column_null :quotes, :anime_id, false
+    change_column_null :quotes, :user_id, false
+    change_column_null :quotes, :content, false
+
+    add_foreign_key :quotes, :users
+    add_foreign_key :quotes, :anime
+    add_foreign_key :quotes, :characters
```

Comment:
is this correct?
```
  # Delete all entries with incomplete dataset
    Quote.where(anime_id: nil).destroy_all

    # Merge anime_id data over to media_id
    rename_column :quotes, :anime_id, :media_id

    # Add type for polymorphic relationship
    add_column :quotes, :media_type, :string,
    change_column_null :quotes, :media_type, false, default: 'Anime'

    # Update all existing entries
    Quote.update_all(media_type: 'Anime')

    # Delete all entries with incomplete dataset
    Quote.where(user_id: nil).destroy_all

    # Makes sure no more invalid data can be added
    change_column_null :quotes, :user_id, false

    # Add reference to characters table
    add_reference :quotes, :character, foreign_key: true
  end
```
What I don't get is why are we only adding foreign key to character?